src/test/ui/repr/repr-align-assign.fixed

```diff
@@ -2,11 +2,11 @@
 
 #![allow(dead_code)]
 
-#[repr(align(8))] //~ ERROR incorrect `repr(align)` attribute format
+#[repr(align=8)] //~ ERROR incorrect `repr(align)` attribute format
```

Comment:
I have no idea why but I can't make this fix to work. The suggestion is machine applicable, shows in stderr but for whatever reason doesn't show up in suggestions after I added a `dbg` call:
```patch
diff --git a/src/tools/compiletest/src/runtest.rs b/src/tools/compiletest/src/runtest.rs
--- a/src/tools/compiletest/src/runtest.rs	(revision 4319f72eca7f720d458355f517f59502fa986dd7)
+++ b/src/tools/compiletest/src/runtest.rs	(date 1622939470653)
@@ -3313,6 +3313,7 @@
                 },
             )
             .unwrap();
+            dbg!(&suggestions);
             let fixed_code = apply_suggestions(&unfixed_code, &suggestions).unwrap_or_else(|_| {
                 panic!("failed to apply suggestions for {:?} with rustfix", self.testpaths.file)
             });
```